pkg/corerp/frontend/controller/applications/testdata/graph-app-directroute-out.json

```diff
@@ -2,7 +2,7 @@
     {
```

Comment:
@nithyatsu @rynowak Please carefully review these test cases.

Reply:
This one is right.